public/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8">
+    <title>Color Match Game</title>
+        <link href="/css/styles.css" rel="stylesheet" />
+  </head>
+<body>
+<h1>Color Drop</h1>
+<br>
+<div id="container">
+  <form name="playerform"><label for="key">Player Name: </label><br />
```

Comment:
This form element is never closed, which means the rest of the document ends up being inside the form. When the reset button is clicked the first time, because it's a button which is technically inside the form, and the submit action of the form isn't handled in code anywhere, it ends up posting the form as a GET action. This puts the name value of the other input in the form onto the url as the query-string. The form won't let you re-submit once this happens unless the value in the name input changes. Properly closing the form earlier (before the reset button) would prevent this. However, there's another issue (see next comment.)